src/GovUk.Education.ExploreEducationStatistics.Common/Model/Either.cs

```diff
@@ -178,6 +178,19 @@ public static class EitherTaskExtensions
 
             return await func();
         }
+        
+        [Obsolete("Use OnSuccessDo or OnSuccessVoid for chaining a non-generic Task")]
+        public static async Task<Either<TFailure, Unit>> OnSuccess<TFailure, TSuccess1>(
```

Comment:
Would suggest we write some basic tests for this, although it's going to end up throwing warnings about using obsolete methods?

Reply:
Yeah it would do - I'd hope that we don't need to write any tests for it cos we'll never use it!